src/components/authenticate/authenticate.test.js

```diff
@@ -70,7 +70,7 @@ describe('Authenticate', () => {
   });
 
   it('should render InfoParagraph with appropriate message', () => {
-    expect(wrapper.find('InfoParagraph').text()).to.include(`You are looking into a saved account. In order to ${props.nextAction} you need to enter your passphrase`);
+    expect(wrapper.find('InfoParagraph').text(`You are looking into a saved account. In order to ${props.nextAction} you need to enter your passphrase`));
```

Comment:
The assertion should be 
`expect(wrapper.find('InfoParagraph')).to.have.text(...` 
the current test won't fail if the text is changed.

Reply:
change done